cdap-runtime-ext-dataproc/src/test/java/io/cdap/cdap/runtime/spi/common/DataprocUtilsTest.java

```diff
@@ -55,5 +55,19 @@ public void testParseLabelsWithoutVal() {
     expected.put("k1", "");
     expected.put("k2", "");
     Assert.assertEquals(expected, DataprocUtils.parseKeyValueConfig("k1,k2=", ",", "="));
+    Map<String, String> got = DataprocUtils.parseKeyValueConfig("", ",", "=");
```

Comment:
can remove this line

Reply:
Removed. It was used for debugging, forgot to remove it.